src/CMakeLists.txt

```diff
@@ -246,9 +246,13 @@ else()
 blt_add_executable(
   NAME raja-perf.exe
   SOURCES RAJAPerfSuiteDriver.cpp
+  INCLUDES ${PROJECT_BINARY_DIR}/include
   DEPENDS_ON ${RAJA_PERFSUITE_EXECUTABLE_DEPENDS}
   )
 install( TARGETS raja-perf.exe
          RUNTIME DESTINATION bin
        )
 endif()
+
+#blt_print_target_properties(TARGET raja-perf.exe)
```

Comment:
It would be good to add a comment above this line that it is there for debugging if needed.